<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml">
  <head>
    <meta charset="utf-8" />
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
    <meta name="generator" content="pandoc" />
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <meta name="author" content="Brian M. Schilder" />
    <meta name="date" content="2023-01-30" />
    <title>Brian M. Schilder’s resume</title>

    
        <link href="CV_files/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
        <link href="CV_files/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />
        <link href="CV_files/paged-0.20/css/resume.css" rel="stylesheet" />
        <script src="CV_files/paged-0.20/js/config.js"></script>
        <script src="CV_files/paged-0.20/js/paged.js"></script>
        <script src="CV_files/paged-0.20/js/hooks.js"></script>
    
        <link rel="stylesheet" href="css/override.css" type="text/css" />
    
  </head>
  <body>

<!-- Tutorial:  https://towardsdatascience.com/create-your-resume-with-pagedown-package-in-r-123ca6310d52 -->
<!-- pagedown Issues: https://github.com/rstudio/pagedown/issues -->
<!-- datadrivencv package: https://github.com/nstrayer/datadrivencv -->
<!-- textnets vignette: https://sicss.io/2020/materials/day3-text-analysis/text-networks/rmarkdown/Text_Networks.html -->
<!-- Only show sidebar on first page: https://github.com/rstudio/pagedown/issues/147 -->
<style type="text/css">
.pagedjs_page:not(:first-of-type) {
  --sidebar-width: 1rem;
  /*--sidebar-background-color: #ffffff;*/
  --main-width: calc(var(--content-width) - var(--sidebar-width));
  --decorator-horizontal-margin: 0.2in;
}
</style>
<div id="aside" class="section level1">
<h1>Aside</h1>
<!-- ![Brian M. Schilder](./images/wildfire_circle.png){width="100%" style="border-radius:10%"} -->
<div style="display:inline; float:right; width:50%;">

</div>
<div id="contact" class="section level2">
<h2>Contact</h2>
<h4>
<i class='fa fa-envelope'></i> <a href="mailto:brian_schilder@alumni.brown.edu" class="email">brian_schilder@alumni.brown.edu</a><br><i class='fa fa-linkedin'></i> <a href="https://www.linkedin.com/in/brian-schilder">LinkedIn</a><br><i class='fa fa-github'></i> <a href="https://github.com/bschilder">GitHub</a><br><i class='fa fa-twitter'></i> <a href="https://twitter.com/BMSchilder">Twitter</a><br><i class='fa fa-globe'></i> <a href="https://bschilder.github.io/BMSchilder">Personal Website</a><br><i class='fa fa-globe'></i> <a href="https://www.neurogenomics.co.uk/">Lab Website</a><br><i class='fa fa-phone'></i> US: <a href='tel:+1 908-268-9859'>+1 908-268-9859</a><br><i class='fa fa-phone'></i> UK: <a href='tel:+44 073-0653-7736'>+44 073-0653-7736</a>
</h4>
</div>
<div id="summary" class="section level2">
<h2>Summary</h2>
<h4>
<p><i class='fa fa-suitcase'></i> <a href="#experience">14+ years of research experience.</a></p>
<p><i class='fa fa-file'></i> <a href="#publications">18 peer-reviewed publications to date.</a></p>
<p><i class='fa fa-desktop'></i> <a href="#software">35 bioinformatics tools developed.</a></p>
<i class='fa fa-chalkboard-teacher'></i> <a href="#teaching">12+ years of teaching/supervising experience.</a>
</h4>
</div>
<div id="toc" class="section level2">
<h2>Table of Contents</h2>
<h4>
<i class='fa fa-graduation-cap'></i> <a href="#education">Education</a><br><i class='fa fa-check'></i> <a href="#skills">Skills</a><br><i class='fa fa-file'></i> <a href="#publications">Publications</a><br><i class='fa fa-file'></i> <a href="#preprints">Preprints</a><br><i class='fa fa-book'></i> <a href="#acknowledgements">Acknowledgements</a><br><i class='fa fa-suitcase'></i> <a href="#experience">Experience</a><br><i class='fa fa-chalkboard-teacher'></i> <a href="#teaching">Teaching</a><br><i class='fa fa-desktop'></i> <a href="#software">Software</a><br><i class='fa fa-desktop'></i> <a href="#webapps">Webapps</a><br><i class='fa fa-desktop'></i> <a href="#websites">Websites</a><br><i class='fa fa-dollar'></i> <a href="#grants">Grants</a><br><i class='fa fa-award'></i> <a href="#awards">Awards</a>
</h4>
<hr>
<p><em>This resume was made with
<a href="https://github.com/rstudio/pagedown"><code>pagedown</code></a>.</em></p>
<p><em>Last updated: Jan-30-2023</em></p>
</div>
</div>
<div id="main" class="section level1">
<h1>Main</h1>
<div id="title" class="section level2">
<h2>Brian M. Schilder</h2>
<p>Passionately pursuing multi-disciplinary research to improve human lives.</p>
<div id="phd-candidate" class="section level3">
<h3>PhD Candidate</h3>
</div>
<div id="scb-mphil" class="section level3">
<h3>ScB, MPhil</h3>
<!-- ### Affiliations {#affiliations} -->
<!-- ```{r, results='asis', echo=FALSE} -->
<!-- parse_profile(types = "affiliation") -->
<!-- ``` -->
</div>
</div>
<div id="education" class="section level2" data-icon="graduation-cap" data-concise="true">
<h2>Education</h2>
<div id="imperial-college-london" class="section level3">
<h3>Imperial College London</h3>
<p><strong>PhD</strong>: Clinical Medical Research; Computational Neurogenomics</p>
<p><a href='https://www.google.com/maps/place/London,+UK'>London, UK</a></p>
<p>2024</p>
<p><strong>Thesis</strong>: Multi-omic medicine: dissecting the cell-type-specific molecular mechanisms underlying neurodegenerative disease genomics</p>
</div>
<div id="the-george-washington-university" class="section level3">
<h3>The George Washington University</h3>
<p><strong>MPhil</strong>: Human Paleobiology; Evolutionary Neuroscience &amp; Genomics</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2017</p>
<p><strong>Thesis</strong>: The evolution of the hippocampus and adult neurogenesis: Novel insights into the origins of human memory</p>
</div>
<div id="brown-university" class="section level3">
<h3>Brown University</h3>
<p><strong>ScB</strong>: Cognitive Neuroscience; Neurological Diseases &amp; Disorders</p>
<p><a href='https://www.google.com/maps/place/Providence,+RI,+USA'>Providence, RI, USA</a></p>
<p>2011</p>
</div>
</div>
<div id="skills" class="section level2" data-icon="check">
<h2>Core Skills</h2>
<div id="research" class="section level3">
<h3>Research</h3>
<p><a href="#experience"><strong>14</strong></a>+ years of highly multidisciplinary research experience. Creative and rapid hypothesis generation, project design, and problem solving.</p>
<p>N/A</p>
<div class="concise">
<ul>
<li><strong>Fields</strong>: Including but not limited to: Neuroscience, systems biology, bioinformatics, machine learning, multi-omics, biomedicine, cognition, statistics, computer science, phylogenetics, paleoanthropology biomechanics, primatology, histology/microscropy.</li>
<li><strong>Publications</strong>: Strong writing skills as evidenced by <a href="#publications"><strong>18</strong></a> peer-reviewed scientific publications, 17 international conference posters and <a href="#grants"><strong>10</strong></a> sucessful grant applications.</li>
</ul>
</div>
<p>N/A</p>
</div>
<div id="programming" class="section level3">
<h3>Programming</h3>
<p>Extensive experience in developing highly reproducible scripts and software packages to interrogate large and diverse data.</p>
<p>N/A</p>
<div class="concise">
<ul>
<li><strong>Bioinformatics</strong>: Including but not limited to: GWAS, QTL, bulk/scRNA-seq &amp; epigenomics, machine learning, high-performance computing, GitHub, GitHub Actions, Docker/Singularity containers, DockerHub, conda, Nextflow, reproducibility.</li>
<li><strong>R</strong>: Created <a href="#software"><strong>26</strong></a> R packages to date, including on Bioconductor and CRAN. Experienced in created Shiny Apps and Rmarkdown reports.</li>
<li><strong>Python</strong>: Regularly uses ML packages (e.g. sklearn, tensorflow, Keras). Developed and own IP for PubReporter, a software for extracting and conducting topic modelling/NLP on relevant scientific literature at scale.</li>
<li><strong>Javascript/HTML/CSS</strong>: Created <a href="#webapps"><strong>10</strong></a>+ websites, web apps, and interactive reports. Developed templates to automatically render websites on-the-fly from CSVs using Javascript.</li>
</ul>
</div>
<p>N/A</p>
</div>
<div id="communication" class="section level3">
<h3>Communication</h3>
<p>Effective and engaging scientific presentations to a variety of audiences.</p>
<p>N/A</p>
<div class="concise">
<ul>
<li><strong>Peers</strong>: Extensive experience presenting research proposals/findings to both internal and external labs/conferences. Invited speaker at leading research institutions. Diverse, multi-disciplinary and global collaborative network.</li>
<li><strong>Students</strong>: Teaching assistant and/or guest lecturer for courses in diverse fields (biological anthropology, neuroscience, computational biology).</li>
<li><strong>Public</strong>: Developed and delivered numerous community engagement programs for a wide variety of stakeholders, including students (preschool through postgraduate), community members, and patients (e.g. Children’s National Hospital, Parkinson’s UK).</li>
</ul>
</div>
<p>N/A</p>
</div>
<div id="project-management" class="section level3">
<h3>Project Management</h3>
<p>Coordinates multiple independent projects at once with researchers of varying degrees of education and research experience. Effective handling of many projects at once.</p>
<p>N/A</p>
<div class="concise">
<ul>
<li><strong>Documentation</strong>: Detailed and understandable in-code documentation is second-nature.</li>
<li><strong>Version control</strong>: Extensive and daily use of git, GitHub Issues, GitHub Projects.</li>
<li><strong>Supervision</strong>: Supervises numerous student research projects.</li>
</ul>
</div>
<p>N/A</p>
</div>
</div>
<div id="publications" class="section level2" data-icon="file">
<h2>Publications</h2>
<div id="dissecting-the-shared-genetic-architecture-of-suicide-attempt-psychiatric-disorders-and-known-risk-factors" class="section level3">
<h3>Dissecting the Shared Genetic Architecture of Suicide Attempt, Psychiatric Disorders, and Known Risk Factors</h3>
<p><em>Biological Psychiatry</em> (2022) 91(3):313-327; <a href="https://doi.org/10.1016/j.biopsych.2021.05.029" class="uri">https://doi.org/10.1016/j.biopsych.2021.05.029</a></p>
<p>N/A</p>
<p>2022</p>
<p>N Mullins, J Kang, AI Campos,…<strong>BM Schilder</strong>, et al. </p>
</div>
<div id="genetic-analysis-of-the-human-microglial-transcriptome-across-brain-regions-aging-and-disease-pathologies" class="section level3">
<h3>Genetic analysis of the human microglial transcriptome across brain regions, aging and disease pathologies</h3>
<p><em>Nature Genetics</em> (2022) <a href="https://doi.org/10.1038/s41588-021-00976-y" class="uri">https://doi.org/10.1038/s41588-021-00976-y</a></p>
<p>N/A</p>
<p>2022</p>
<p>K de Paiva Lopes, G JL Snijders, J Humphrey, A Allan, M Sneeboer, E Navarro, <strong>BM Schilder</strong>…T Raj</p>
</div>
<div id="multi-omic-insights-into-parkinsons-disease-from-genetic-associations-to-functional-mechanisms" class="section level3">
<h3>Multi-omic insights into Parkinson’s Disease: From genetic associations to functional mechanisms</h3>
<p><em>Neurobiology of Disease</em> (2021) 105580; <a href="https://doi.org/10.1016/j.nbd.2021.105580" class="uri">https://doi.org/10.1016/j.nbd.2021.105580</a></p>
<p>N/A</p>
<p>2021</p>
<p><strong>BM Schilder</strong>, E Navarro, T Raj</p>
</div>
<div id="fine-mapping-of-parkinsons-disease-susceptibility-loci-identifies-putative-causal-variants" class="section level3">
<h3>Fine-Mapping of Parkinson’s Disease Susceptibility Loci Identifies Putative Causal Variants</h3>
<p><em>Human Molecular Genetics</em> (2021) ddab294; <a href="https://doi.org/10.1093/hmg/ddab294" class="uri">https://doi.org/10.1093/hmg/ddab294</a></p>
<p>N/A</p>
<p>2021</p>
<p><strong>BM Schilder</strong>, T Raj</p>
</div>
<div id="echolocator-an-automated-end-to-end-statistical-and-functional-genomic-fine-mapping-pipeline" class="section level3">
<h3>echolocatoR: An Automated End-to-End Statistical and Functional Genomic Fine-Mapping Pipeline</h3>
<p><em>Bioinformatics</em> (2021) btab658; <a href="https://doi.org/10.1093/bioinformatics/btab658" class="uri">https://doi.org/10.1093/bioinformatics/btab658</a></p>
<p>N/A</p>
<p>2021</p>
<p><strong>BM Schilder</strong>, J Humphrey, T Raj</p>
</div>
<div id="mungesumstats-a-bioconductor-package-for-the-standardisation-and-quality-control-of-many-gwas-summary-statistics" class="section level3">
<h3>MungeSumstats: A Bioconductor Package for the Standardisation and Quality Control of Many GWAS Summary Statistics</h3>
<p><em>Bioinformatics</em> (2021) 37(23):4593-4596; <a href="https://doi.org/10.1093/bioinformatics/btab665" class="uri">https://doi.org/10.1093/bioinformatics/btab665</a></p>
<p>N/A</p>
<p>2021</p>
<p>A Murphy, <strong>BM Schilder</strong>, NG Skene</p>
</div>
<div id="dysregulation-of-mitochondrial-and-proteo-lysosomal-genes-in-parkinsons-disease-myeloid-cells" class="section level3">
<h3>Dysregulation of mitochondrial and proteo-lysosomal genes in Parkinson’s disease myeloid cells</h3>
<p><em>Nature Genetics</em> (2021) <a href="https://doi.org/10.1101/2020.07.20.212407" class="uri">https://doi.org/10.1101/2020.07.20.212407</a></p>
<p>N/A</p>
<p>2021</p>
<p>E Navarro, E Udine, K de Paiva Lopes, M Parks, G Riboldi, <strong>BM Schilder</strong>…T Raj</p>
</div>
<div id="phenome-wide-and-eqtl-associations-of-covid-19-genetic-risk-loci" class="section level3">
<h3>Phenome-wide and eQTL Associations of COVID-19 Genetic Risk Loci</h3>
<p><em>iScience</em> (2021) <a href="https://doi.org/10.1016/j.isci.2021.102550" class="uri">https://doi.org/10.1016/j.isci.2021.102550</a></p>
<p>N/A</p>
<p>2021</p>
<p>C Moon, <strong>BM Schilder</strong>, T Raj, K-l Huang</p>
</div>
<div id="genome-wide-association-study-of-over-40000-bipolar-disorder-cases-provides-novel-biological-insights" class="section level3">
<h3>Genome-Wide Association Study of over 40,000 Bipolar Disorder Cases Provides Novel Biological Insights</h3>
<p><em>Nature Genetics</em> (2020) 53:817-829; <a href="https://doi.org/10.1038/s41588-021-00857-4" class="uri">https://doi.org/10.1038/s41588-021-00857-4</a></p>
<p>N/A</p>
<p>2020</p>
<p>N Mullins, AJ Forstner, KS O’Connell, B Coombes, JRI Coleman…<strong>BM Schilder</strong>… et al.</p>
</div>
<div id="tensor-decomposition-of-stimulated-monocyte-and-macrophage-gene-expression-profiles-identifies-neurodegenerative-disease-specific-trans-eqtls" class="section level3">
<h3>Tensor decomposition of stimulated monocyte and macrophage gene expression profiles identifies neurodegenerative disease-specific trans-eQTLs</h3>
<p><em>PLOS Genetics</em> (2020) 16(9):e1008549; <a href="https://doi.org/10.1101/499509" class="uri">https://doi.org/10.1101/499509</a></p>
<p>N/A</p>
<p>2020</p>
<p>S Ramdhani, E Navarro, E Udine, AG Efthymiou, <strong>BM Schilder</strong>, M Parks, A Goate, T Raj</p>
</div>
<div id="evolutionary-shifts-dramatically-reorganized-the-human-hippocampal-complex" class="section level3">
<h3>Evolutionary shifts dramatically reorganized the human hippocampal complex</h3>
<p><em>Journal of Comparative Neurology</em> (2019) 528(17):3143-3170; <a href="https://doi.org/10.1002/cne.24822" class="uri">https://doi.org/10.1002/cne.24822</a></p>
<p>N/A</p>
<p>2019</p>
<p><strong>BM Schilder</strong>, HM Petry, PR Hof</p>
</div>
<div id="fairshake-toolkit-to-evaluate-the-findability-accessibility-interoperability-and-reusability-of-research-digital-resources." class="section level3">
<h3>FAIRshake: Toolkit to Evaluate the Findability, Accessibility, Interoperability, and Reusability of Research Digital Resources.</h3>
<p><em>Cell Systems</em> (2019) 9; <a href="https://doi.org/10.1016/j.cels.2019.09.011" class="uri">https://doi.org/10.1016/j.cels.2019.09.011</a></p>
<p>N/A</p>
<p>2019</p>
<p>D Clarke, L Wang, A Jones, M Wojciechowicz, D Torre, K Jagodnik, S Jenkins, P McQuilton, Z Flamholz, M Silverstein, <strong>BM Schilder</strong>…A Ma’ayan</p>
</div>
<div id="geneshot-search-engine-for-ranking-genes-from-arbitrary-text-queries" class="section level3">
<h3>Geneshot: search engine for ranking genes from arbitrary text queries</h3>
<p><em>Nucleic Acids Research</em> (2019) 47(W1):W571-W577; <a href="https://doi.org/10.1093/nar/gkz393" class="uri">https://doi.org/10.1093/nar/gkz393</a></p>
<p>N/A</p>
<p>2019</p>
<p>A Lachmann, <strong>BM Schilder</strong>, ML Wojciechowicz, D Torre, MV Kuleshov, AB Keenan, A Ma’ayan</p>
</div>
<div id="expression2kinases-x2k-web-linking-expression-signatures-to-upstream-cell-signaling-networks" class="section level3">
<h3>eXpression2Kinases (X2K) Web: linking expression signatures to upstream cell signaling networks</h3>
<p><em>Nucleic Acids Research</em> (2018) 46(W1):W171-W179; <a href="https://doi.org/10.1093/nar/gky458" class="uri">https://doi.org/10.1093/nar/gky458</a></p>
<p>N/A</p>
<p>2018</p>
<p>DJB Clarke, MV Kuleshov, <strong>BM Schilder</strong>, D Torre, ME Duffy, AB Keenan, A Lachmann, AS Feldmann, GW Gundersen, MC Silverstein, Z Wang</p>
</div>
<div id="defining-elemental-imitation-mechanisms-a-comparison-of-cognitive-and-motor-spatial-imitation-learning-across-object--and-computer-based-tasks" class="section level3">
<h3>Defining elemental imitation mechanisms: A comparison of cognitive and motor-spatial imitation learning across object- and computer-based tasks</h3>
<p><em>Journal of Cognition and Development</em> (2015) 17(2):221-243; <a href="https://doi.org/10.1080/15248372.2015.1053483" class="uri">https://doi.org/10.1080/15248372.2015.1053483</a></p>
<p>N/A</p>
<p>2015</p>
<p>F Subiaul, L Zimmerman, E Renner, <strong>BM Schilder</strong>, R Barr</p>
</div>
<div id="take-the-monkey-and-run" class="section level3">
<h3>Take the monkey and run</h3>
<p><em>Journal of Neuroscience Methods</em> (2015) 248:28-31; <a href="http://doi.org/10.1016/j.jneumeth.2015.03.023" class="uri">http://doi.org/10.1016/j.jneumeth.2015.03.023</a></p>
<p>N/A</p>
<p>2015</p>
<p>KA Phillips, MK Hambright, K Hewes, <strong>BM Schilder</strong>, CN Ross, SD Tardif</p>
</div>
<div id="becoming-a-high-fidelity---super---imitator-what-are-the-contributions-of-social-and-individual-learning" class="section level3">
<h3>Becoming a high-fidelity - super - imitator: what are the contributions of social and individual learning?</h3>
<p><em>Developmental Science</em> (2014) 18(6):1025-1035; <a href="http://doi.org/10.1111/desc.12276" class="uri">http://doi.org/10.1111/desc.12276</a></p>
<p>N/A</p>
<p>2014</p>
<p>F Subiaul, EM Patterson, <strong>BM Schilder</strong>, E Renner, R Barr</p>
</div>
<div id="working-memory-constraints-on-imitation-and-emulation" class="section level3">
<h3>Working memory constraints on imitation and emulation</h3>
<p><em>Journal of Experimental Child Psychology</em> (2014) 128:190-200; <a href="http://doi.org/10.1016/j.jecp.2014.07.005" class="uri">http://doi.org/10.1016/j.jecp.2014.07.005</a></p>
<p>N/A</p>
<p>2014</p>
<p>F Subiaul, <strong>BM Schilder</strong></p>
</div>
</div>
<div id="preprints" class="section level2" data-icon="file">
<h2>Preprints</h2>
<div id="the-rworkflows-suite-automated-continuous-integration-for-quality-checking-documentation-website-creation-and-containerised-deployment-of-r-packages" class="section level3">
<h3>The rworkflows suite: automated continuous integration for quality checking, documentation website creation, and containerised deployment of R packages</h3>
<p><em>Research Square</em> (2023) <a href="https://doi.org/10.21203/rs.3.rs-2399015/v1" class="uri">https://doi.org/10.21203/rs.3.rs-2399015/v1</a></p>
<p>N/A</p>
<p>2023</p>
<p><strong>BM Schilder</strong>, AE Murphy, NG Skene</p>
</div>
<div id="cuttag-recovers-up-to-half-of-encode-chip-seq-peaks" class="section level3">
<h3>CUT&amp;Tag recovers up to half of ENCODE ChIP-seq peaks</h3>
<p><em>bioRxiv</em> (2022) <a href="https://doi.org/10.1101/2022.03.30.486382" class="uri">https://doi.org/10.1101/2022.03.30.486382</a></p>
<p>N/A</p>
<p>2022</p>
<p>D Hu, L Abbasova, <strong>BM Schilder</strong>, A Nott, NG Skene, SJ Marzi</p>
</div>
<div id="epicompare-r-package-for-the-comparison-and-quality-control-of-epigenomic-peak-files" class="section level3">
<h3>EpiCompare: R package for the comparison and quality control of epigenomic peak files</h3>
<p><em>bioRxiv</em> (2022) <a href="https://doi.org/10.1101/2022.07.22.501149" class="uri">https://doi.org/10.1101/2022.07.22.501149</a></p>
<p>N/A</p>
<p>2022</p>
<p>S Choi, <strong>BM Schilder</strong>, L Abbasova, AE Murphy, NG Skene</p>
</div>
</div>
<div id="acknowledgements" class="section level2" data-icon="book">
<h2>Acknowledgements</h2>
<div id="eqtl-catalogue-a-compendium-of-uniformly-processed-human-gene-expression-and-splicing-qtls." class="section level3">
<h3>eQTL Catalogue: a compendium of uniformly processed human gene expression and splicing QTLs.</h3>
<p><em>Nature Genetics</em> (2021) 53:1290-1299; <a href="https://doi.org/10.1038/s41588-021-00924-w" class="uri">https://doi.org/10.1038/s41588-021-00924-w</a></p>
<p>N/A</p>
<p>2021</p>
<p>N Kerimov, JD Hayhurst, K Peikova et al.</p>
</div>
<div id="functionally-informed-fine-mapping-and-polygenic-localization-of-complex-trait-heritability" class="section level3">
<h3>Functionally-informed fine-mapping and polygenic localization of complex trait heritability</h3>
<p><em>Nature Genetics</em> (2020) <a href="https://doi.org/10.1038/s41588-020-00735-5" class="uri">https://doi.org/10.1038/s41588-020-00735-5</a></p>
<p>N/A</p>
<p>2020</p>
<p>O Weissbrod…AL Price</p>
</div>
<div id="wayfinding-the-science-and-mystery-of-how-humans-navigate-the-world." class="section level3">
<h3>Wayfinding: The science and mystery of how humans navigate the world.</h3>
<p><em>St. Martin’s Press</em> (2019) ISBN-13: 978-1250096968; <a href="https://www.amazon.co.uk/Wayfinding-Science-Mystery-Humans-Navigate/dp/1250096960" class="uri">https://www.amazon.co.uk/Wayfinding-Science-Mystery-Humans-Navigate/dp/1250096960</a></p>
<p>N/A</p>
<p>2019</p>
<p>MR O’Connor</p>
</div>
<div id="eeg-oscillations-reveal-neural-correlates-of-evidence-accumulation" class="section level3">
<h3>EEG oscillations reveal neural correlates of evidence accumulation</h3>
<p><em>Frontiers in Decision Neuroscience</em> (2012) 6(106):Jan-13; <a href="https://doi.org/10.3389/fnins.2012.00106" class="uri">https://doi.org/10.3389/fnins.2012.00106</a></p>
<p>N/A</p>
<p>2012</p>
<p>M van Vugt, P Simen, L Nystrom, P Holmes, J Cohen</p>
</div>
<div id="trial-by-trial-adaptation-of-decision-making-performance-a-model-based-eeg-analysis" class="section level3">
<h3>Trial-by-trial adaptation of decision making performance: a model-based EEG analysis</h3>
<p><em>Interdisciplinary Perspectives on Cognition, Education, and the Brain</em> (2011) 7;</p>
<p>N/A</p>
<p>2011</p>
<p>M van Vugt, P Simen, J Cohen</p>
</div>
</div>
<div id="experience" class="section level2" data-icon="suitcase">
<h2>Research Experience</h2>
<div id="phd-candidate-1" class="section level3">
<h3>PhD Candidate</h3>
<p>Imperial College London (UK Dementia Research Institute, Department of Brain Sciences)</p>
<p><a href='https://www.google.com/maps/place/London,+UK'>London, UK</a></p>
<p>2024 - 2020</p>
<ul>
<li>Thesis: Multi-omic medicine: dissecting the cell-type-specific molecular mechanisms underlying neurodegenerative disease genomics.</li>
<li>Objective: Computationally dissect the subtraits, cell-types underlying the genomic signatures of Alzheimer’s Disease and Parkinson’s Disease.</li>
</ul>
</div>
<div id="consultant" class="section level3">
<h3>Consultant</h3>
<p>120/80 MKTG</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>- - 2019</p>
<ul>
<li>Offers consultation services to a wide portfolio of high-profile digital healthcare companies, with a focus on data analytics, research publication, and strategic transparency.</li>
<li>Provides data-driven business intelligence reports using proprietary software scientific literature mining software that I have created.</li>
<li>Professional company web design.</li>
</ul>
</div>
<div id="bioinformatician-ii" class="section level3">
<h3>Bioinformatician II</h3>
<p>Icahn School of Medicine at Mount Sinai (Department of Neuroscience / Department of Neurology / Department of Genetics &amp; Genomics / Ronald M. Loeb Center for Alzheimer’s Disease)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2020 - 2018</p>
<ul>
<li>Developed machine learning systems to integrate large-scale multi-omics datasets (e.g. whole-genome sequencing, bulk and single-cell RNA-seq, epigenomics, clinical data) to uncover the molecular mechanisms underlying neurodegenerative diseases (e.g. Alzheimer’s, Parkinson’s, ALS).</li>
<li>Computationally identified specific disease-causal variants, pathways and cell-types for subsequent functional wet lab validation (e.g. CRISPR-cas9 editing in patient-derived cell cultures, iPSCs and cerebral organoids).</li>
</ul>
</div>
<div id="bioinformatician-ii-1" class="section level3">
<h3>Bioinformatician II</h3>
<p>Icahn School of Medicine at Mount Sinai (Department of Pharmacological Sciences)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2018 - 2017</p>
<ul>
<li>Conducted computational systems biology research. Integrated and analyzed large-scale genomic and biomedical data (e.g. Python, R, JavaScript).</li>
<li>Developed evolutionary algorithm to optimize gene network kinase regulator prediction (eXpression2Kinases).</li>
<li>Developed and deployed computational tools, software, databases and web applications for basic and clinical research, resulting in 3 peer-reviewed publications.</li>
</ul>
</div>
<div id="participant" class="section level3">
<h3>Participant</h3>
<p>Technische Universität Dresden / eMed (Summer School in Systems Medicine)</p>
<p><a href='https://www.google.com/maps/place/Frauenchiemsee,+Germany'>Frauenchiemsee, Germany</a></p>
<p>2017</p>
<ul>
<li>Attended lectures and extended skills in extraction and analysis of big data from biomedical and neurogenomic resources.</li>
<li>Developed, performed and wrote manuscript for collaborative bioinformatics research project in less than one week.</li>
</ul>
</div>
<div id="participant-1" class="section level3">
<h3>Participant</h3>
<p>Icahn School of Medicine at Mount Sinai (Scientific Computing &amp; Data Science)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2016</p>
<ul>
<li>Intensive summer school in high-performance computing, coding, genome database utilization and bioinformatics methods including transcriptomics and genetic association testing.</li>
</ul>
</div>
<div id="graduate-student" class="section level3">
<h3>Graduate Student</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2017 - 2013</p>
<ul>
<li>Thesis: Evolution of the Hippocampus and Adult Neurogenesis Research</li>
<li>Investigated the evolution of human-specific cognitive abilities and neurological disease susceptibilities (e.g. Alzheimer’s Disease).</li>
<li>Focused on neuroanatomical, transcriptomic and genomic evolution of the human hippocampus and memory.</li>
</ul>
</div>
<div id="collaborator" class="section level3">
<h3>Collaborator</h3>
<p>Trinity University / Southwestern National Primate Research Center (Department of Neuroscience)</p>
<p><a href='https://www.google.com/maps/place/San+Antonio,+TX,+USA'>San Antonio, TX, USA</a></p>
<p>2017 - 2014</p>
<ul>
<li>Investigated the neurobiological mechanisms underlying the ameliorating effects of exercise on relapse-remitting Multiple Sclerosis.</li>
</ul>
</div>
<div id="teaching-assistant-project-leader" class="section level3">
<h3>Teaching Assistant / Project Leader</h3>
<p>The George Washington University / Rutgers University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Ileret,+Kenya'>Ileret, Kenya</a></p>
<p>2014</p>
<ul>
<li>Served as Teaching Assistant while excavating Lower Paleolithic hominin sites (Homo, Paranthropus).</li>
<li>As Project Leader, investigated the running biomechanics of local Daasanach tribespeople while mentoring undergraduate students.</li>
</ul>
</div>
<div id="research-assistant" class="section level3">
<h3>Research Assistant</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2013 - 2011</p>
<ul>
<li>Performed dissection, histology, microscopy and quantitative stereology in post-mortem primate brain tissues.</li>
<li>Trained junior and senior personnel on lab protocols.</li>
</ul>
</div>
<div id="senior-lab-manager" class="section level3">
<h3>Senior Lab Manager</h3>
<p>The George Washington University (Department of Speech, Language &amp; Hearing Sciences)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2013 - 2011</p>
<ul>
<li>Organized and trained dozens of undergraduates to conduct weekly cognitive development research; designed and/or directly contributed to over 15 research projects in two years.</li>
</ul>
</div>
<div id="volunteer-researcher" class="section level3">
<h3>Volunteer Researcher</h3>
<p>University of Winnipeg / University of Belgrade (Department of Anthropology / Department of Archaeology)</p>
<p><a href='https://www.google.com/maps/place/Sićevo,+Serbia'>Sićevo, Serbia</a></p>
<p>2012</p>
<ul>
<li>Excavated Paleolithic fossils and tools (H. heidelbergensis, H. neanderthalensis) at Mala Balanica, Velika Balanica, and Pešturina sites.</li>
</ul>
</div>
<div id="volunteer-researcher-1" class="section level3">
<h3>Volunteer Researcher</h3>
<p>Universidad de Murcia (Department of Zoology &amp; Physical Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Murcia,+Spain'>Murcia, Spain</a></p>
<p>2011</p>
<ul>
<li>Excavated Paleolithic fossils and tools from Cueva Negra (H. heidelbergensis) and Sima de las Palomas (H. neanderthalensis) with an international research team.</li>
</ul>
</div>
<div id="volunteer-research-intern" class="section level3">
<h3>Volunteer Research Intern</h3>
<p>American Museum of Natural History (Division of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2011</p>
<ul>
<li>Contributed to paleoanthropological research on primate fossils using 3D morphometry imaging equipment including Minolta, Microscribe and CT.</li>
</ul>
</div>
<div id="paid-research-intern" class="section level3">
<h3>Paid Research Intern</h3>
<p>Princeton University (Princeton Neuroscience Institute)</p>
<p><a href='https://www.google.com/maps/place/Princeton,+NJ,+USA'>Princeton, NJ, USA</a></p>
<p>2010</p>
<ul>
<li>Investigated the neural basis of decision-making in humans.</li>
<li>Recruited participants, recorded EEG and analyzed data in MATLAB.</li>
</ul>
</div>
<div id="student-researcher" class="section level3">
<h3>Student Researcher</h3>
<p>Brown University (Department of Cognitive, Linguistic &amp; Psychological Sciences)</p>
<p><a href='https://www.google.com/maps/place/Providence,+RI,+USA'>Providence, RI, USA</a></p>
<p>2010 - 2009</p>
<ul>
<li>Experimental Analysis of Animal Behavior &amp; Cognition: Conducted various operant conditioning experiments on rats. Gained experience in animal behavioral training, data collection, and data analysis in MATLAB.</li>
<li>Laboratory in Genes and Behavior: Tested transgenic mice with modified N-type voltage-gated calcium channel subunits in a battery of cognitive and sensorimotor tasks. Results were published.</li>
</ul>
</div>
</div>
<div id="teaching" class="section level2" data-icon="chalkboard-teacher">
<h2>Teaching / Mentoring Experience</h2>
<div id="research-mentor" class="section level3">
<h3>Research Mentor</h3>
<p>Imperial College London (Department of Brain Sciences / Department of Life Sciences)</p>
<p><a href='https://www.google.com/maps/place/London,+UK'>London, UK</a></p>
<p>- - 2020</p>
<ul>
<li>Mentored students and affiliated projects:</li>
<li>Kitty Murphy (PhD): ‘Evolutionary pressures on cell types: leveraging species differences to gain insight into neurodegenerative disease risk’</li>
<li>Sheen Lei (BSc): ‘Benchmarking cell-type-specific enrichment of genome-wide disease signatures’</li>
<li>Ted Reese (MSc): ‘Computational cell-type annotation of single-cell epigenomics data’</li>
<li>Xindong Sun (MSc): ‘Benchmark of Targeted insertion of promoters sequencing (TIP-seq) on histone modification H3K27ac and H3K27me3 in K562 cell line’</li>
<li>Shuhan Shen (MSc): ‘Evaluation and optimisation of methods for identifying the cell types underlying genetic disease signatures’</li>
<li>Lusheng Li (MSc): ‘Genetic identification of cell types underlying mammalian phenotypes’</li>
<li>Sera Choi (BSc): ‘EpiCompare: R package for QC and benchmarking epigenetic datasets’</li>
<li>Emilie Cottard (MSc) &amp; Will Lunt (BSc): ‘A meta-analysis of selective cell-type vulnerability in Parkinson’s Disease neuropathology’</li>
<li>Jai Chapman (BSc): ‘Expression Weighted Cell Type Enrichment as a Tool for Identifying Cell Types Underlying Rare Disease Phenotypes’</li>
<li>Bobby Gordon-Smith (MSc): ‘Identification of cell types involved in rare disease-associated human phenotypes’</li>
<li>Leyla Abbasova (MSc): ‘Analysis and optimisation of CUT&amp;Tag for epigenomic profiling of the brain’</li>
<li>Barney Hill (BSc): ‘Identification of cell-types associated with latent factors inferred from phenome-wide GWAS summary statistics’</li>
</ul>
</div>
<div id="research-mentor-1" class="section level3">
<h3>Research Mentor</h3>
<p>Icahn School of Medicine at Mount Sinai (Department of Neuroscience / Department of Neurology / Department of Genetics &amp; Genomics / Ronald M. Loeb Center for Alzheimer’s Disease)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2020 - 2019</p>
<ul>
<li>Mentored MS, MD, and PhD students in projects focused on computational exploration of phenotype clustering and genomic regulation of neurodegenerative diseases.</li>
</ul>
</div>
<div id="research-co-mentor" class="section level3">
<h3>Research Co-mentor</h3>
<p>Icahn School of Medicine at Mount Sinai (Department of Pharmacological Sciences)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2018</p>
<ul>
<li>Mentored students and affiliated projects:</li>
<li>Vivian Utti (BSc): ‘ChEA3: Transcription Factor Enrichment Analysis’ as part of the Summer Research Training Program in Biomedical Big Data Science.</li>
<li>Mary Duffy (PhD): ‘Predicting upstream kinase regulators from interaction network databases’</li>
<li>Zach Flamholz (BSc): ‘modEnrichr: a suite of gene set enrichment analysis tools for model organisms’</li>
</ul>
</div>
<div id="guest-lecturer" class="section level3">
<h3>Guest Lecturer</h3>
<p>Icahn School of Medicine at Mount Sinai (Department of Pharmacological Sciences)</p>
<p><a href='https://www.google.com/maps/place/New+York,+NY,+USA'>New York, NY, USA</a></p>
<p>2018</p>
<ul>
<li>Lectured on data visualization in Python and Jupyter notebooks in the PhD/MD course ‘Programming for Big Data Biomedicine’.</li>
</ul>
</div>
<div id="research-mentor-2" class="section level3">
<h3>Research Mentor</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2017 - 2016</p>
<ul>
<li>Mentored students and affiliated projects:</li>
<li>Jamie Kleiner (BSc): ‘Animal model simulating MS and exercise’s impact on adult hippocampal neurogenesis’</li>
</ul>
</div>
<div id="teaching-assistant" class="section level3">
<h3>Teaching Assistant</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2015</p>
<ul>
<li>Course: ““Human Brain Evolution””</li>
<li>Guest lectured, graded all assignments and exams, and provided additional educational support during office hours.</li>
</ul>
</div>
<div id="teaching-assistant-1" class="section level3">
<h3>Teaching Assistant</h3>
<p>The George Washington University (Department of Psychology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2014</p>
<ul>
<li>Course: ““Biological Psychology””</li>
<li>Led undergraduates in article discussions, graded all assignments and exams, and provided additional educational support during office hours.</li>
</ul>
</div>
<div id="teaching-assistant-2" class="section level3">
<h3>Teaching Assistant</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2014 - 2013</p>
<ul>
<li>Course: ““Biological Anthropology””</li>
<li>Led undergraduate students in two, 2-hour lab sessions per week, graded lab assignments and exams, and provided additional educational support during office hours.</li>
</ul>
</div>
<div id="research-mentor-3" class="section level3">
<h3>Research Mentor</h3>
<p>The George Washington University (Department of Psychology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2013 - 2012</p>
<ul>
<li>Mentored students and affiliated projects:</li>
<li>Anushka Gokhale (BSc): ‘Infants’ Social Assessment of Characters Through Eye Gaze’</li>
</ul>
</div>
<div id="lab-protocol-trainer" class="section level3">
<h3>Lab Protocol Trainer</h3>
<p>The George Washington University (Department of Anthropology)</p>
<p><a href='https://www.google.com/maps/place/Washington,+DC,+USA'>Washington, DC, USA</a></p>
<p>2013 - 2011</p>
<ul>
<li>Trained undergraduate, graduate, and post-doctoral researchers in Social Cognition Lab and Lab for Evolutionary Neuroscience in a variety of methodological research protocols.</li>
</ul>
</div>
</div>
<div id="software" class="section level2" data-icon="desktop">
<h2>Software Packages</h2>
<div id="rworkflows" class="section level3">
<h3>rworkflows</h3>
<p>Continuous integration for R packages. Automates testing, documentation website building, and containerised deployment<br><a href='https://github.com/neurogenomics/rworkflows' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/rworkflows</a><br><a href='https://doi.org/10.21203/rs.3.rs-2399015/v1' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.21203/rs.3.rs-2399015/v1</a></p>
<p>N/A</p>
<p>1.</p>
</div>
<div id="tipseeker" class="section level3">
<h3>TIPseeker</h3>
<p>R package for post-processing [single-cell] TIP-seq data<br><a href='https://github.com/neurogenomics/TIPseeker' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/TIPseeker</a><br></p>
<p>N/A</p>
<p>2.</p>
</div>
<div id="peakyfinders" class="section level3">
<h3>PeakyFinders</h3>
<p>R package for mining, calling, and importing epigenomic peaks<br><a href='https://github.com/neurogenomics/PeakyFinders' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/PeakyFinders</a><br></p>
<p>N/A</p>
<p>3.</p>
</div>
<div id="skillnet" class="section level3">
<h3>SkillNet</h3>
<p>Creates user-specific contribution networks from GitHub Organization repositories<br><a href='https://github.com/neurogenomics/SkillNet' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/SkillNet</a><br></p>
<p>N/A</p>
<p>4.</p>
</div>
<div id="phenomix" class="section level3">
<h3>phenomix</h3>
<p>R package for the exploration and analysis of many genotype-phenotype datasets at once<br><a href='https://github.com/neurogenomics/phenomix' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/phenomix</a><br></p>
<p>N/A</p>
<p>5.</p>
</div>
<div id="magma_celltyping" class="section level3">
<h3>MAGMA_Celltyping</h3>
<p>Identify cell types underlying the associations found in GWAS summary statistics<br><a href='https://github.com/neurogenomics/MAGMA_Celltyping' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/MAGMA_Celltyping</a><br></p>
<p>N/A</p>
<p>6.</p>
</div>
<div id="ewce" class="section level3">
<h3>EWCE</h3>
<p>Expression Weighted Celltype Enrichment<br><a href='https://github.com/NathanSkene/EWCE' target='_blank'><i class='fa fa-github'></i> https://github.com/NathanSkene/EWCE</a><br></p>
<p>N/A</p>
<p>7.</p>
</div>
<div id="epicompare" class="section level3">
<h3>EpiCompare</h3>
<p>R package for QC and benchmarking epigenetic datasets<br><a href='https://github.com/neurogenomics/EpiCompare' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/EpiCompare</a><br></p>
<p>N/A</p>
<p>8.</p>
</div>
<div id="multiewce" class="section level3">
<h3>MultiEWCE</h3>
<p>R package for analysing multiple gene lists using EWCE<br><a href='https://github.com/neurogenomics/MultiEWCE' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/MultiEWCE</a><br></p>
<p>N/A</p>
<p>9.</p>
</div>
<div id="hpoexplorer" class="section level3">
<h3>HPOExplorer</h3>
<p>Functions for working with the Human Phenotype Ontology data<br><a href='https://github.com/neurogenomics/HPOExplorer' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/HPOExplorer</a><br></p>
<p>N/A</p>
<p>10.</p>
</div>
<div id="orthogene" class="section level3">
<h3>orthogene</h3>
<p>Interspecies gene mapping<br><a href='https://github.com/neurogenomics/orthogene' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/orthogene</a><br></p>
<p>N/A</p>
<p>11.</p>
</div>
<div id="mungesumstats" class="section level3">
<h3>MungeSumstats</h3>
<p>Standardise the format of summary statistics from GWAS<br><a href='https://github.com/neurogenomics/MungeSumstats' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/MungeSumstats</a><br><a href='https://doi.org/10.1093/bioinformatics/btab665' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1093/bioinformatics/btab665</a></p>
<p>N/A</p>
<p>12.</p>
</div>
<div id="scnlp" class="section level3">
<h3>scNLP</h3>
<p>Tools for applying natural language processing (NLP) techniques to single-cell (sc) omics data<br><a href='https://github.com/neurogenomics/scNLP' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/scNLP</a><br></p>
<p>N/A</p>
<p>13.</p>
</div>
<div id="sckirby" class="section level3">
<h3>scKirby</h3>
<p>Automated ingestion and conversion of various single-cell data formats<br><a href='https://github.com/neurogenomics/scKirby' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/scKirby</a><br></p>
<p>N/A</p>
<p>14.</p>
</div>
<div id="templater" class="section level3">
<h3>templateR</h3>
<p>Template for developing R packages<br><a href='https://github.com/neurogenomics/templateR' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/templateR</a><br><a href='https://doi.org/10.21203/rs.3.rs-2399015/v1' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.21203/rs.3.rs-2399015/v1</a></p>
<p>N/A</p>
<p>15.</p>
</div>
<div id="echolocator" class="section level3">
<h3>echolocatoR</h3>
<p>R package for end-to-end statistical and functional fine-mapping with extensive dataset access<br><a href='https://github.com/RajLabMSSM/echolocatoR' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echolocatoR</a><br><a href='https://doi.org/10.1093/bioinformatics/btab658' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1093/bioinformatics/btab658</a></p>
<p>N/A</p>
<p>16.</p>
</div>
<div id="echodata" class="section level3">
<h3>echodata</h3>
<p>Examples of fine-mapped GWAS summary statistics, data formatting functions, and API access to the echolocatoR Fine-mapping Portal<br><a href='https://github.com/RajLabMSSM/echodata' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echodata</a><br></p>
<p>N/A</p>
<p>17.</p>
</div>
<div id="echoannot" class="section level3">
<h3>echoannot</h3>
<p>Functions for annotating genomic data with annotations and epigenomic data<br><a href='https://github.com/RajLabMSSM/echoannot' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echoannot</a><br></p>
<p>N/A</p>
<p>18.</p>
</div>
<div id="echoplot" class="section level3">
<h3>echoplot</h3>
<p>R package for LocusZoom-inspired GWAS/QTL visualization, with API access to LD panels<br><a href='https://github.com/RajLabMSSM/echoplot' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echoplot</a><br></p>
<p>N/A</p>
<p>19.</p>
</div>
<div id="echoconda" class="section level3">
<h3>echoconda</h3>
<p>Various utility functions to find, build, and use conda environments from within R <br><a href='https://github.com/RajLabMSSM/echoconda' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echoconda</a><br></p>
<p>N/A</p>
<p>20.</p>
</div>
<div id="echotabix" class="section level3">
<h3>echotabix</h3>
<p>Tabix indexing and querying<br><a href='https://github.com/RajLabMSSM/echotabix' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echotabix</a><br></p>
<p>N/A</p>
<p>21.</p>
</div>
<div id="echold" class="section level3">
<h3>echoLD</h3>
<p>LD downloading and processing.<br><a href='https://github.com/RajLabMSSM/echoLD' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echoLD</a><br></p>
<p>N/A</p>
<p>22.</p>
</div>
<div id="echodeps" class="section level3">
<h3>echodeps</h3>
<p>Creates interactive dependency networks for R packages.<br><a href='https://github.com/RajLabMSSM/echodeps' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echodeps</a><br></p>
<p>N/A</p>
<p>23.</p>
</div>
<div id="echogithub" class="section level3">
<h3>echogithub</h3>
<p>Access and process metadata from GitHub<br><a href='https://github.com/RajLabMSSM/echogithub' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/echogithub</a><br></p>
<p>N/A</p>
<p>24.</p>
</div>
<div id="downloadr" class="section level3">
<h3>downloadR</h3>
<p>Single- and multi-threaded downloading functions<br><a href='https://github.com/RajLabMSSM/downloadR' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/downloadR</a><br></p>
<p>N/A</p>
<p>25.</p>
</div>
<div id="cataloguer" class="section level3">
<h3>catalogueR</h3>
<p>R package for rapid API-access and colocalization of summary statistics from eQTL Catalogue<br><a href='https://github.com/RajLabMSSM/catalogueR' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/catalogueR</a><br></p>
<p>N/A</p>
<p>26.</p>
</div>
<div id="pubreporter" class="section level3">
<h3>PubReporter</h3>
<p>Proprietary Python package for extracting relevant scientific literature and citations, performing topic modelling, and generating interactive business intelligence reports.<br><br></p>
<p>N/A</p>
<p>27.</p>
</div>
</div>
<div id="webapps" class="section level2" data-icon="desktop">
<h2>Web Apps</h2>
<div id="geneshot" class="section level3">
<h3>Geneshot</h3>
<p>Flexible tool to identify genes associated with any biomedical term and to predict novel target genes<br><a href='http://amp.pharm.mssm.edu/geneshot' target='_blank'><i class='fa fa-link'></i> http://amp.pharm.mssm.edu/geneshot</a><br><a href='https://doi.org/10.1093/nar/gkz393' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1093/nar/gkz393</a></p>
<p>N/A</p>
<p>1.</p>
</div>
<div id="x2k" class="section level3">
<h3>X2K</h3>
<p>eXpression 2 Kinases (X2K) Web: Automated computational pipeline to infer kinase regulators from weighted or unweighted gene lists<br><a href='http://amp.pharm.mssm.edu/X2K' target='_blank'><i class='fa fa-link'></i> http://amp.pharm.mssm.edu/X2K</a><br><a href='https://doi.org/10.1093/nar/gky458' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1093/nar/gky458</a></p>
<p>N/A</p>
<p>2.</p>
</div>
<div id="rare-disease-celltyping-apps" class="section level3">
<h3>Rare Disease Celltyping Apps</h3>
<p>Web portal connecting to multiple R Shiny apps to explore, visualize, and download cell type-specific enrichment results and systematically prioritised gene targets for over 6,000 rare disease phenotypes.<br><a href='https://github.com/neurogenomics/rare_disease_celltyping_apps' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/rare_disease_celltyping_apps</a><br></p>
<p>N/A</p>
<p>3.</p>
</div>
<div id="parkinsons-disease-omics-review" class="section level3">
<h3>Parkinson’s Disease Omics Review</h3>
<p>Data and code associated with the Parkinson’s Disease review paper by Schilder, Navarro &amp; Raj (Neurobiology of Disease, 2021)<br><a href='https://github.com/RajLabMSSM/PD_omics_review' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/PD_omics_review</a><br><a href='https://rajlabmssm.github.io/PD_omics_review/' target='_blank'><i class='fa fa-link'></i> https://rajlabmssm.github.io/PD_omics_review/</a><br><a href='https://doi.org/10.1016/j.nbd.2021.105580' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1016/j.nbd.2021.105580</a></p>
<p>N/A</p>
<p>4.</p>
</div>
<div id="selective-vulnerability-meta-analysis" class="section level3">
<h3>Selective Vulnerability Meta-analysis</h3>
<p>Selective Vulnerability Meta-analysis: Shiny app dedicated to the exploration and dissemination of meta-analysed cell counts manually curated and harmonised from the Parkinson’s Disease literature<br><a href='https://github.com/neurogenomics/SelectiveVulnerabilityMetaAnalysis' target='_blank'><i class='fa fa-github'></i> https://github.com/neurogenomics/SelectiveVulnerabilityMetaAnalysis</a><br></p>
<p>N/A</p>
<p>5.</p>
</div>
<div id="echolocator-fine-mapping-portal" class="section level3">
<h3>echolocatoR Fine-mapping Portal</h3>
<p>Access to interactive plots and fine-mapping results across many GWAS/QTL datasets using echolocatoR<br><a href='https://github.com/RajLabMSSM/Fine_Mapping_Shiny' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/Fine_Mapping_Shiny</a><br><a href='https://rajlab.shinyapps.io/Fine_Mapping_Shiny' target='_blank'><i class='fa fa-link'></i> https://rajlab.shinyapps.io/Fine_Mapping_Shiny</a><br><a href='https://doi.org/10.1093/bioinformatics/btab658' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1093/bioinformatics/btab658</a></p>
<p>N/A</p>
<p>6.</p>
</div>
<div id="tensor-decomposition-shiny-app" class="section level3">
<h3>Tensor Decomposition Shiny App</h3>
<p>Interactive application to explore and download all results and plots from Ramdhani et al. (PLOS Genetics, 2020)<br><a href='https://github.com/RajLabMSSM/Tensor_myeloid' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/Tensor_myeloid</a><br><a href='https://rajlab.shinyapps.io/Tensor_myeloid' target='_blank'><i class='fa fa-link'></i> https://rajlab.shinyapps.io/Tensor_myeloid</a><br><a href='https://doi.org/10.1101/499509' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1101/499509</a></p>
<p>N/A</p>
<p>7.</p>
</div>
<div id="hippocampal-evolution" class="section level3">
<h3>Hippocampal Evolution</h3>
<p>Interactive code, results and visualization for the manuscript “Evolutionary selective pressures dramatically expanded and reorganized the human hippocampal complex”<br><a href='https://github.com/bschilder/Hippo_Eco' target='_blank'><i class='fa fa-github'></i> https://github.com/bschilder/Hippo_Eco</a><br><a href='https://bschilder.github.io/Hippo_Eco/HPsubfield_eco' target='_blank'><i class='fa fa-link'></i> https://bschilder.github.io/Hippo_Eco/HPsubfield_eco</a><br><a href='https://doi.org/10.1002/cne.24822' target='_blank'><i class='fa fa-file'></i> https://doi.org/10.1002/cne.24822</a></p>
<p>N/A</p>
<p>8.</p>
</div>
</div>
<div id="websites" class="section level2" data-icon="desktop">
<h2>Websites</h2>
<div id="personal-website" class="section level3">
<h3>Personal Website</h3>
<p><br><a href='https://github.com/bschilder/BMSchilder' target='_blank'><i class='fa fa-github'></i> https://github.com/bschilder/BMSchilder</a><br><a href='https://bschilder.github.io/BMSchilder' target='_blank'><i class='fa fa-link'></i> https://bschilder.github.io/BMSchilder</a><br></p>
<p>N/A</p>
<p>1.</p>
</div>
<div id="official-raj-lab-website" class="section level3">
<h3>Official Raj Lab Website</h3>
<p><br><a href='https://github.com/RajLabMSSM/RajLab_website' target='_blank'><i class='fa fa-github'></i> https://github.com/RajLabMSSM/RajLab_website</a><br><a href='http://www.rajlab.org' target='_blank'><i class='fa fa-link'></i> http://www.rajlab.org</a><br></p>
<p>N/A</p>
<p>2.</p>
</div>
</div>
<div id="grants" class="section level2" data-icon="dollar">
<h2>Grants</h2>
<div id="total-all-grants-2949052-total-as-primary-applicant-310562" class="section level3">
<h3><strong>Total (all grants)</strong>: $2,949,052<br><strong>Total (as primary applicant)</strong>: $310,562</h3>
<p>N/A</p>
<p>N/A</p>
<hr>
<p>N/A</p>
</div>
<div id="imperial-uk-research-institute-impact-acceleration-account-imperial-college-london" class="section level3">
<h3>Imperial UK Research Institute Impact Acceleration Account,<br>Imperial College London</h3>
<p><strong>Project</strong>: ‘Creating commercial kit solutions for single cell epigenetic profiling of histone marks and transcription factors’</p>
<p>N/A</p>
<p>2023</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Co-applicant</li>
<li><strong>PI</strong>: NG Skene</li>
<li><strong>Amount</strong>: £80,000</li>
</ul>
</div>
</div>
<div id="turing-community-award-alan-turing-institute" class="section level3">
<h3>Turing Community Award,<br>Alan Turing Institute</h3>
<p><strong>Project</strong>: ‘Multi-omic medicine: dissecting the cell-type-specific molecular mechanisms underlying neurodegenerative disease genomics’</p>
<p>N/A</p>
<p>2024 - 2022</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: £3,000</li>
</ul>
</div>
</div>
<div id="national-institutes-of-health" class="section level3">
<h3>National Institutes of Health</h3>
<p><strong>Project</strong>: ‘Statistical and functional fine-mapping of bipolar disorder genetic risk loci’</p>
<p>N/A</p>
<p>2022</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Co-applicant</li>
<li><strong>PI</strong>: N Mullins</li>
</ul>
</div>
</div>
<div id="collaborative-single-cell-and-spatial-transcriptomics-studies-award-programme-uk-dementia-research-institute" class="section level3">
<h3>Collaborative Single Cell and Spatial Transcriptomics Studies Award Programme,<br>UK Dementia Research Institute</h3>
<p><strong>Project</strong>: ‘Amplifying genome coverage of single cell epigenetic profiling of the human brain’</p>
<p>N/A</p>
<p>2021</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Co-applicant</li>
<li><strong>PI</strong>: D Hu, NG Skene</li>
<li><strong>Amount</strong>: £12,790</li>
</ul>
</div>
</div>
<div id="national-institutes-of-health-1" class="section level3">
<h3>National Institutes of Health</h3>
<p><strong>Project</strong>: “Cognitive Systems Analysis of Alzheimer’s Disease Genetic and Phenotypic Data”</p>
<p>N/A</p>
<p>2020</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Co-applicant</li>
<li><strong>PI</strong>: T Raj, D Knowles</li>
<li><strong>Amount</strong>: $2,523,431</li>
</ul>
</div>
</div>
<div id="uk-dementia-research-institute" class="section level3">
<h3>UK Dementia Research Institute</h3>
<p><strong>Project</strong>: ‘UK DRI at Imperial Distinguished Studentship’</p>
<p>N/A</p>
<p>2024 - 2020</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: £217,000</li>
</ul>
</div>
</div>
<div id="the-michael-j.-fox-foundation" class="section level3">
<h3>The Michael J. Fox Foundation</h3>
<p><strong>Project</strong>: “The Role of Peripheral Myeloid Cells in Parkinson’s Disease”</p>
<p>N/A</p>
<p>2019 - 2017</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Fundee</li>
<li><strong>PI</strong>: T Raj</li>
</ul>
</div>
</div>
<div id="the-michael-j.-fox-foundation-1" class="section level3">
<h3>The Michael J. Fox Foundation</h3>
<p><strong>Project</strong>: ‘Functional Fine-Mapping of LRRK2 Locus’</p>
<p>N/A</p>
<p>2020 - 2017</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Fundee</li>
<li><strong>PI</strong>: T Raj</li>
</ul>
</div>
</div>
<div id="national-science-foundation" class="section level3">
<h3>National Science Foundation</h3>
<p><strong>Project</strong>: ‘The evolution of the hippocampus and adult neurogenesis: novel insights into the origins of human memory’</p>
<p>N/A</p>
<p>2017</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: $31 543</li>
</ul>
</div>
</div>
<div id="wenner-gren" class="section level3">
<h3>Wenner-Gren</h3>
<p><strong>Project</strong>: ‘The evolution of the hippocampus and adult neurogenesis: insights into the origins of human memory’</p>
<p>N/A</p>
<p>2017</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: $19,512</li>
</ul>
</div>
</div>
<div id="leakey-foundation" class="section level3">
<h3>Leakey Foundation</h3>
<p><strong>Project</strong>: ‘The evolution of the hippocampus and adult neurogenesis: Novel insights into the origins of human memory’</p>
<p>N/A</p>
<p>2016</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: $15,000</li>
</ul>
</div>
</div>
<div id="cosmos-club" class="section level3">
<h3>COSMOS Club</h3>
<p><strong>Project</strong>: ‘The evolution of adult neurogenesis across primates’</p>
<p>N/A</p>
<p>2016</p>
<div class="concise">
<ul>
<li><strong>Role</strong>: Primary applicant</li>
<li><strong>PI</strong>: BM Schilder</li>
<li><strong>Amount</strong>: $3,250</li>
</ul>
</div>
</div>
</div>
<div id="awards" class="section level2" data-icon="award">
<h2>Awards</h2>
<div id="prize-for-computational-reproducibility-in-dementia-research-honourable-mention-uk-dementia-research-institute" class="section level3">
<h3>Prize for Computational Reproducibility in Dementia Research (honourable mention),<br>UK Dementia Research Institute</h3>
<p><strong>Project</strong>: ‘MungeSumstats: A Bioconductor package for the standardisation and quality control of many GWAS summary statistics’</p>
<p>N/A</p>
<p>2022</p>
<div class="concise">
<p><a href="https://demondementia.com/neurohack2022">Individually awarded</a> for outstanding contributions during the NEUROHACK 2022, a competitive 4-day hackathon to apply AI in finding ALS therapeutic solutions.</p>
</div>
</div>
<div id="award-for-outstanding-contribution-neurohack-deep-dementia-phenotyping-network-demon" class="section level3">
<h3>Award for Outstanding Contribution,<br>NEUROHACK, Deep Dementia Phenotyping Network (DEMON)</h3>
<p><strong>Project</strong>: ‘Predicting ALS drug targets using integrative multi-modal deep learning’</p>
<p>N/A</p>
<p>2022</p>
<div class="concise">
<p><a href="https://t.co/3gLMoqFQLa">Jointly awarded inaugural prize</a> with Kitty Murphy.</p>
</div>
</div>
<div id="prize-for-computational-reproducibility-in-dementia-research-uk-dementia-research-institute" class="section level3">
<h3>Prize for Computational Reproducibility in Dementia Research,<br>UK Dementia Research Institute</h3>
<p><strong>Project</strong>: ‘echolocatoR: an automated end-to-end statistical and functional genomic fine-mapping pipeline’</p>
<p>N/A</p>
<p>2021</p>
<div class="concise">

</div>
</div>
<div id="centre-photography-competition-uk-dementia-research-institute" class="section level3">
<h3>Centre Photography Competition,<br>UK Dementia Research Institute</h3>
<p><strong>Project</strong>: ‘Wildfire Circle, Golden Brain, Wildfire, Geneshot, Geology of Biology, Neon Brain’</p>
<p>N/A</p>
<p>2021</p>
<div class="concise">
<p><a href="https://www.imperial.ac.uk/dementia-research-institute/seminars-and-events/events/centre-photography-competition/">One of the winners of the scientific image competition</a>.</p>
</div>
</div>
<div id="art-of-the-brain-friedman-brain-institute-icahn-school-of-medicine" class="section level3">
<h3>Art of the Brain,<br>Friedman Brain Institute, Icahn School of Medicine</h3>
<p><strong>Project</strong>: ‘Wildfire’</p>
<p>N/A</p>
<p>2019</p>
<div class="concise">
<p><a href="https://friedmanbrain.icahn.mssm.edu/art-of-the-brain">Awarded 2nd place</a> and Featured on the <a href="https://www.biologicalpsychiatryjournal.com/issue/S0006-3223(19)X0012-8">cover of Biological Psychiatry: Volume 87, Issue 12 (2020)</a>. Exhibited and auctioned at the Grady Alexis Gallery (New York City), where all proceeds were voluntarily donated to the Diversity in Neuroscience Initiative.</p>
</div>
</div>
</div>
</div>

<script>
(function() {
  var i, s, j, el, els;
  var create_el = function(type, className, content, isHTML) {
    var el = document.createElement(type);
    if (className) el.className = className;
    if (content) {
      if (isHTML) {el.innerHTML = content;} else {el.innerText = content;}
    }
    return el;
  }

  // replace h2 title with h1
  var title = document.getElementById('title').querySelector('h2');
  title.parentNode.replaceChild(create_el('h1', false, title.innerHTML, true), title);

  // add the class .aside to #aside
  el = document.getElementById('aside');
  if (el) el.className += ' aside';

  // tweak class names of sections, and add icons
  els = document.getElementById('main').querySelectorAll('.level2');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (i === 0 && el.id === 'title') continue;
    el.className += ' main-block';
    if (el.dataset['concise']) el.className += ' concise';
    // if there is no icon, add an icon:
    if (el.firstElementChild.firstChild && el.firstElementChild.firstChild.nodeName !== 'I') {
      s = el.dataset['icon'] || 'bookmark';
      el.firstElementChild.insertBefore(create_el('i', 'fa fa-' + s), el.firstElementChild.firstChild);
    }
  }

  // tweak class names of blocks in sections
  els = document.getElementById('main').querySelectorAll('.level3');
  for (i = 0; i < els.length; i++) {
    el = els[i];
    if (el.parentNode.id === 'title') continue;
    el.className += ' blocks';
    el.innerHTML = '<div class="date"></div>' + '<div class="decorator"></div>' +
      '<div class="details"><header></header><div></div></div>' + el.innerHTML;
    var ps = el.querySelectorAll('p');
    // move the date paragraph to the date div
    if (ps.length >= 3) {
      s = ps[2].innerText;
      s = s === 'N/A' ? [] : s.split(' - ');
      el.removeChild(ps[2]);
      for (j = 0; j < s.length && j < 2; j++) {
        el.children[0].appendChild(create_el('span', false, s[j]))
      }
    }
    // move title, description, location to the details div
    (function(h) {
      h.appendChild(el.children[3]);
      var p = el.children[3];  // description
      if (p.innerText !== 'N/A') {
        h.appendChild(create_el('span', 'place', p.innerHTML, true));
      }
      el.removeChild(p);
      p = el.children[3];  // location
      if (p.innerText !== 'N/A') {
        s = create_el('span', 'location', p.innerHTML, true);
        s.innerHTML = '<i class="fa fa-map-marker-alt"></i> ' + s.innerHTML;
        h.appendChild(s);
      }
      el.removeChild(p);
    })(el.children[2].firstElementChild);
    // move the rest of content in a block into the last div of the details div
    s = el.children[2];
    while (j = s.nextSibling) {
      if (j.className === 'aside') { s = j; continue; }
      el.children[2].lastElementChild.append(j);
    }
  }
})();
</script>


  </body>
</html>
